.golangci.yml

```diff
@@ -0,0 +1,73 @@
+run:
+  concurrency: 4
+  deadline: 1m
+  issues-exit-code: 1
+  tests: true
+  # build-tags:
+  #   - mytag
+  skip-dirs:
+    - static
+  skip-files:
+    - .*_mock.go
+    - jail/doc.go
+
+output:
+  format: colored-line-number
+  print-issued-lines: true
+  print-linter-name: true
+
+linters-settings:
+  errcheck:
+    check-type-assertions: false
+    check-blank: false
+  govet:
+    check-shadowing: false
+  golint:
+    min-confidence: 0.8
+  gofmt:
+    simplify: true
+  gocyclo:
+    min-complexity: 16
```

Comment:
not `max-complexity`?

Reply:
I took this from their sample: https://github.com/golangci/golangci-lint/blob/master/.golangci.example.yml#L31
I guess it means minimum complexity for the linter to complain.